server/aws-lsp-codewhisperer/src/language-server/agenticChat/agenticChatResultStream.ts

```diff
@@ -183,6 +183,10 @@ export class AgenticChatResultStream {
 
         return {
             write: async (intermediateChatResult: ChatMessage) => {
+                const isLoading = intermediateChatResult.messageId?.startsWith('loading-')
```

Comment:
could be worth mentioning this behavior in a doc string. Not obvious to the caller that messageIds prefixed by `loading-` are handled separately.

Reply:
Sure, updating